[DOC beta] Mark `LinkComponent#activeClass` as public.

Serabe opened the pull request:
`activeClass` is mark as private in `Ember.LinkComponent` though in the
documentation suggests to use it for modifying the class added when link is
active.

homu commented:
:umbrella: The latest upstream changes (presumably #14039) made this pull request unmergeable. Please resolve the merge conflicts.

Serabe committed:
[DOC beta] Mark `LinkComponent#activeClass` as public.

`activeClass` is mark as private in `Ember.LinkComponent` though in the
documentation suggests to use it for modifying the class added when link is
active.

Serabe commented:
Fixed!

mixonic commented:
Thanks @Serabe!
